tools/accuracy_checker/accuracy_checker/adapters/detection_head.py

```diff
@@ -0,0 +1,159 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import math
+from collections import namedtuple
+import numpy as np
+
+from ..adapters import Adapter
+from ..config import NumberField, ListField, ConfigError
+from ..representation import DetectionPrediction
+
+DetectionLayerOutput = namedtuple('DetectionLayerOutput', ['prob_name', 'reg_name', 'anchor_index', 'anchor_size', 'win_scale', 'win_length', 'win_trans_x', 'win_trans_y'])
+
+class HeadDetectionAdapter(Adapter):
+    __provider__ = 'head_detection'
```

Comment:
what the difference between this adapter and FaceDetectionAdapter?
the code looks very similar

Reply:
@hojaedev I see that you reuse metric, it is cool, but I do not see any changes in adapter
Is there is opportunity to reuse code base of FaceDetectionAdapter (e.g. inheriting from it or unite 2 adapter into one)?